### PR TITLE
A user can login to an existing account

### PR DESCRIPTION
**Pull request by angiecstn**
## PT link

https://www.pivotaltracker.com/story/show/160721279

## Included in PR:

* Installing and implementing Devise 
* Configuring Devise attribute
* Adding Devise fields and forms to views
* Creating Devise user model
* Feature testing login functionalities

## What we learnt
* How to use devise

### CODE REVIEW

File: config/initializers/devise.rb [hunk 1]
```diff
@@ -0,0 +1,290 @@
+# frozen_string_literal: true
```
Comment (magnus-thor) [inline config/initializers/devise.rb]: Clean out the comments from this file

**Review by tochman — COMMENTED** [review]
Please reconsider the naming of you step definition file and extract the assertions to a separate file.

**Review by magnus-thor — CHANGES_REQUESTED** [review]
Make sure the PT links is going to the story.

File: config/initializers/devise.rb (hunk 1 above)
Comment (magnus-thor) [reply config/initializers/devise.rb]: And make sure the PT link goes to the story

Comment (angiecstn) [reply config/initializers/devise.rb]: Done, thanks

Comment (magnus-thor) [reply config/initializers/devise.rb]: Please remove the comments from this file and do not just press resolve conversation

**Review by magnus-thor — CHANGES_REQUESTED** [review]

**Review by magnus-thor — APPROVED** [review]
Good job

**Pull request merged**